java/vector/src/main/java/org/apache/arrow/vector/types/pojo/DictionaryEncoding.java

```diff
@@ -28,7 +31,11 @@
   private final boolean ordered;
   private final Int indexType;
 
-  public DictionaryEncoding(long id, boolean ordered, Int indexType) {
+  @JsonCreator
+  public DictionaryEncoding(
+      @JsonProperty("id") long id,
+      @JsonProperty("isOrdered") boolean ordered,
```

Comment:
That would work a little easier with the annotations, so doesn't make a difference to me.  I was following the naming from the DictionaryEncoding table in schema.fbs - @wesm  is "ordered" is fine with you also?